scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -531,18 +535,18 @@ <h6>No causative variants</h6>
 {% endmacro %}
 
 {% macro suspects_list(suspects, institute, case, manual_rank_options, cancer_tier_options) %}
+   {{ clinvar_howto_modal() }}
   <div class="card mt-3">
-    <div class="d-flex align-items-center" data-bs-toggle='tooltip' title="Displays all variants that has been pinned for this case">
-      <h6><span class="fa fa-map-pin ms-3 mt-2"></span>&nbsp;Pinned variants ({{suspects|length}})
-      </h6>
+    <div class="d-flex align-items-center" data-bs-toggle='tooltip' title="Displays all variants that has been pinned or submitted to ClinVar for this case">
```

Comment:
```suggestion
    <div class="d-flex align-items-center" data-bs-toggle='tooltip' title="Displays all variants that have been pinned or included in a ClinVar submission for this case">
```

Reply:
The wording has been updated, both for the tooltip and header